### DISCUSSION TITLE
Shared Congregation server giving trouble

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#1579] [Q&A] [answered]
I have a situation here that I don't know if it's a bug or the way I'm using it. We have 3 congregations and we use the same email account on the koofr server. What happens: in the weekend meeting, one is Saturday [OK], and two are Sundays (Morning and Evening), however, the media for the Sunday meetings, are the same when added to the congregation's server, I would have to create a koofr server account for each congregation? Currently the alternative here was to put the type of different date for the two congregations not to have the same media.

**Comment by mtdvlpr** [comment, marked as answer]
Hi @e-Feitosa, what you could do is have different subfolders. For example, we have a `[Congregation name]` and a `test` folder in Koofr. I use test when developing. They work as two seperate servers.

**Reply by e-Feitosa** [reply]
Great, I'll do it, if it's not too complicated, a suggestion would be to automatically create a folder in koofr according to the name of the congregation (not sure if that's feasible). But there could be something in the congregation's server settings to differentiate the folder in case two congregations have meetings on the same day on the weekend.
